app/models/early_career_payments/eligibility_answers_presenter.rb

```diff
@@ -0,0 +1,36 @@
+module EarlyCareerPayments
+  class EligibilityAnswersPresenter
+    include ActionView::Helpers::TranslationHelper
+
+    attr_reader :eligibility
+
+    def initialize(eligibility)
+      @eligibility = eligibility
+    end
+
+    # Formats the eligibility as a list of questions and answers, each
+    # accompanied by a slug for changing the answer. Suitable for playback to
+    # the claimant for them to review on the check-your-answers page.
+    #
+    # Returns an array. Each element of this an array is an array of three
+    # elements:
+    # [0]: question text;
+    # [1]: answer text;
+    # [2]: slug for changing the answer.
+    def answers
```

Comment:
There's no  `describe` for this.

Reply:
There is for similar admin classes but not for these, so consistent with those at least. Let come back to this later.